object_storage_buckets.go

```diff
@@ -56,10 +61,11 @@ func (i *ObjectStorageBucket) UnmarshalJSON(b []byte) error {
 
 // ObjectStorageBucketCreateOptions fields are those accepted by CreateObjectStorageBucket
 type ObjectStorageBucketCreateOptions struct {
-	// You may use either Region or Cluster, depending on your account details.
-	Region string `json:"region,omitempty"`
-	// You may use either Region or Cluster, depending on your account details.
+	// Deprecated: Cluster field has been deprecated.
+	// Please consider switching to use the 'Region' field.
+	// If your Cluster is `us-mia-1`, then the region would be `us-mia`.
```

Comment:
Minor wording change to suggest that Cluster will no longer be supported going forward:

```
Deprecated: The Cluster field has been deprecated.

Going forward, the Region field will be the supported way to designate where Object Storage Buckets should be created. For example, a Cluster value of `us-mia-1` will translate to `us-mia` for Region.
```